docs/data-sources/groups.md

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Security"
+---
+# databricks_groups Data Source
+
+Retrieves a list of all [databricks_group](../resources/group.md) display names associated with the workspace, or those matching the provided filter. Maximum 100 results.
+
+
+## Example Usage
+
+Get all groups:
+
+```hcl
+data "databricks_groups" "all" {}
+
+output "all_groups" {
+  value = data.databricksdatabricks_groups.all.display_names
+}
+```
+
+Get groups whose displayName contains `foo` or displayName contains `bar`.:
+
+```hcl
+data "databricks_groups" "this" {
+  filter = "displayName co "foo" or displayName co "bar""
+}
+
+output "foobar_groups" {
+  value = data.databricks_pipelines.this.display_names
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `filter` - (Optional) Query by which the results have to be filtered. Supported operators are equals(`eq`), contains(`co`), starts with(`sw`) and not equals(`ne`).
```

Comment:
We need to link the documentation about filter syntax. Like this: https://docs.databricks.com/api/workspace/groups/list

Reply:
Done